src/taco-cli/cli/run.ts

```diff
@@ -235,6 +244,18 @@ class Run extends commands.TacoCommandBase {
         if (parsedOptions.options["debug"] && parsedOptions.options["release"]) {
             throw errorHelper.get(TacoErrorCodes.ErrorIncompatibleOptions, "--debug", "--release");
         }
+        
+        if (parsedOptions.options["livereload"] && parsedOptions.options["remote"]) {
```

Comment:
Please watch out for merge conflicts with master (after Nisheet's refactoring commit)